docs/data-sources/connectors.md

```diff
@@ -0,0 +1,36 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_connectors Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's documentation https://docs.twingate.com/docs/understanding-access-nodes.
+---
+
+# twingate_connectors (Data Source)
+
+Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/understanding-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_connectors" "all" {}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `connectors` (List of Object) List of connectors (see [below for nested schema](#nestedatt--connectors))
```

Comment:
For consistency, we should capitalize Connectors: `List of Connectors`

Reply:
updated